app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation.js

```diff
@@ -0,0 +1,70 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function SuperNavigationToggle ($module) {
+    this.$module = $module
+    this.$button = _setupButton($module)
+    this.$menu = document.getElementById(this.$button.getAttribute('aria-controls'))
+    this.status = this.$button.getAttribute('aria-expanded') === 'true' ? 'open' : 'closed'
+  }
+
+  function _removeClass (classes, classToRemove) {
+    var arrayOfClasses = classes.split(' ')
+    var classPosition = arrayOfClasses.indexOf(classToRemove)
+    arrayOfClasses.splice(classPosition, 1)
+    return arrayOfClasses.join(' ')
+  }
+
+  function _setupButton (target) {
+    var menuText = 'Menu'
```

Comment:
Could we read this from a data attribute?

Reply:
I've updated the JavaScript to read the text from data attributes: https://github.com/alphagov/govuk_publishing_components/blob/951b14bb286ae910bb3104b2ce61dd017f910eae/app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation-header.js#L10-L12

The data attributes can be then populated from the localisation file.